tools/common/MachO.cs

```diff
@@ -3,13 +3,12 @@
 using System.IO;
 using System.Net;
 using System.Text;
+using Xamarin.Bunder;
```

Comment:
@wachaudh you have a typo :) 

```suggestion
using Xamarin.Bundler;
```